src/cogs/utils.py

```diff
@@ -17,6 +17,13 @@ class Utils(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
+    @commands.command(name="ping")
+    async def ping(self, ctx):
+        #Displays latency between client and bot
```

Comment:
```suggestion
        """Displays latency between client and bot"""
```
Make this into a doc comment so it gets picked up by help